docs/ref/django-admin.txt

```diff
@@ -888,6 +888,10 @@ more robust change detection, and a reduction in power usage.
     for optimal performance. See the `watchman documentation`_ for information
     on how to do this.
 
+    If you get the message ``Error connecting to Watchman: timed out waiting for response``
```

Comment:
I would move this out of the admonition and into some text above.

Reply:
Done!

Reply:
@orf Done!